Validation/MtdValidation/plugins/EtlSimHitsValidation.cc

```diff
@@ -264,7 +540,8 @@ void EtlSimHitsValidation::fillDescriptions(edm::ConfigurationDescriptions& desc
 
   desc.add<std::string>("folder", "MTD/ETL/SimHits");
   desc.add<edm::InputTag>("inputTag", edm::InputTag("mix", "g4SimHitsFastTimerHitsEndcap"));
-  desc.add<double>("hitMinimumEnergy", 0.1);  // [MeV]
+  desc.add<double>("hitMinimumEnergy1Dis", 0.01);   // [MeV]
```

Comment:
again why this change?

Reply:
My mistake, for the old scenario the thresholds should not change.